app/main.py

```
@@ -1,3 +1,89 @@
+
+    def __truediv__(self, other: Any) -> Any:
+        if isinstance(other, (int, float)) and other != 0:
+            return Distance(km=round((self.km / other), 2))
+        elif other == 0:
+            return print("Can't division by 0!")
+        else:
+            raise TypeError(f"Unsupported operand type for / : {type(other)}!")
+
+    def __lt__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int, float)):
+            return self.km < other
+        else:
+            print(f"the {other} is not a Distance and it doesn't a number!")
+            return False
+
+    def __gt__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        if isinstance(other, (int, float)):
+            return self.km > other
+        else:
+            print(f"the {other} is not a Distance and it doesn't a number!")
+            return False
+
+    def __eq__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        if isinstance(other, (int, float)):
+            return self.km == other
+        else:
+            print(f"the {other} is not a Distance and it doesn't a number!")
+            return False
+
+    def __le__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km <= other.km
+        elif isinstance(other, (int, float)):
+            return self.km <= other
+        else:
+            print(f"the {other} is not a Distance and it doesn't a number!")
+            return False
+
+    def __ge__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        if isinstance(other, (int, float)):
+            return self.km >= other
+        else:
+            print(f"the {other} is not a Distance and it doesn't a number!")
+            return False
```

Issue: In the `__ge__` method, for unsupported operand types, you are printing a message and returning `False`. You should raise a `TypeError` for unsupported types instead.